controllers/provisioning_controller.go

```diff
@@ -39,10 +56,29 @@ type ProvisioningReconciler struct {
 // Reconcile updates the cluster settings when the Provisioning
 // resource changes
 func (r *ProvisioningReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-	_ = context.Background()
-	_ = r.Log.WithValues("provisioning", req.NamespacedName)
+	ctx := context.Background()
+	log := r.Log.WithValues("provisioning", req.NamespacedName)
+
+	infra := &osconfigv1.Infrastructure{}
+	err := r.Client.Get(ctx, client.ObjectKey{
+		Name: "cluster",
+	}, infra)
+	if err != nil {
+		log.Info("Unable to determine Platform that the Operator is running on.")
+		return ctrl.Result{}, err
+	}
 
-	// your logic here
+	// Disable ourselves on platforms other than bare metal
+	if infra.Status.Platform != osconfigv1.BareMetalPlatformType {
+		log.Info("Not Baremetal Platform. Setting CBO to disabled state.")
```

Comment:
This log message should include the value of the platform. If you log before testing, then the log message on line 81 isn't needed.

Reply:
Done.